### DISCUSSION TITLE
Error when installing dependencies in python quickstart example

### DISCUSSION THREAD

**Discussion by StianSMN** [databrickslabs/dbx#691] [Q&A]
I am trying to use the python quickstart from a windows vm. I created a venv using python (not conda). from the project folder. I am running python 3.11. When trying to install dependencies, I get an error when installing numba: RuntimeError: Cannot install on Python version 3.11.0; only versions >=3.7,<3.11 are supported. Do I need to use python 3.10 to follow the example

**Comment by renardeinside** [comment]
Hi @StianSMN , 
please provide the issue in the `Issues` not `Discussions`, with relevant commands, proper env description and full error stack trace. Unfortunately without knowing the context its very hard to help you with precise guidance.
